README.md

```diff
@@ -163,15 +167,15 @@ If the second window navigated back once in history, it would resume using the s
 
 #### Concrete Example
 
-If a publisher, say publisher.com, opens a new window on login.com as part of a login flow the second window will be on a transient storage partition and not have access to the same storage as it would if the user navigated directly to login.com on a new window. The two windows will, however, be able to communicate via post message.
+If a publisher, say publisher.example, opens a new window on tracker.example to profile the user the second window will be on a transient storage partition and not have access to the same storage as it would if the user navigated directly to tracker.example on a new window. The two windows will, however, be able to communicate via post message. This prevents usage of the first-party storage partition of tracker.example for tracking that depends on using post message.
 
 ### Proposal 2b: Opt-in restoration of opener
 
 In any cross-origin context where an opener would currently be present, the opener will instead be considered ‘pending’ (not connected) until some heuristic (e.g., user interaction) or specific API (e.g., requestStorageAccess) were called.
 
 ![](./images/rsa_0.png)
 
-If a publisher, say publisher.com, opens a new window on login.com as part of a login flow the two windows could not communicate via post message. If the second window on login.com then met the heuristic threshold or called the relevant API the opener would be restored and communication via post message would be possible.
+If a publisher, say publisher.example, opens a new window on tracker.example as part of tracking the user the two windows could not communicate via post message. If the second window on tracker.example then met the heuristic threshold or called the relevant API the opener would be restored and communication via post message would be possible. This locks down unauthorized cross-window post messaging to prevent usage of first-party storage partitions for unapproved user tracking.
```

Comment:
```suggestion
If a publisher, say publisher.example, opens a new window on tracker.example as part of tracking the user the two windows could not communicate via post message. If the second window on tracker.example then met the heuristic threshold or called the relevant API the opener would be restored and communication via post message would be possible. This locks down unauthorized cross-window post messaging to prevent usage of first-party storage partitions for cross-site tracking.
```

I wanted to replace "unapproved" with "unauthorized" to stay consistent but then figured it's probably not needed at all, the whole thing is pretty clear and suggesting that there's "approved cross-site tracking" adds some unneeded confusion to the sentence, IMO.